deepchem/models/molgan.py

```diff
@@ -122,10 +123,11 @@ def create_generator(self) -> keras.Model:
         The model has two outputs:
             1. edges
             2. nodes
-        The format differs depending on intended use (training or sample generation).
-        For sample generation use flag, sample_generation=True while calling generator
+        The format differs depending on intended use (sample generation or training).
+        For sample generation, flag "sample_generation=True" to be used while calling generator
```

Comment:
The original version uses active voice which is better than passive voice. Can you revert back to active voice?